wg/bsi.md

```diff
@@ -0,0 +1,68 @@
+# Cloud Native Batch System Initiative Working Group Charter
+ 
+## Introduction
+
+Cloud Native Batch System Initiative Working Group is a group formed by 
+passionate batch system maintainers looking to support batch workload in cloud native environments.
+ 
+## Background
+
+Currently, more and more users would like to migrate batch workloads to cloud native environments,
+and multiple existing solutions from different community projects currently address this requirement,
+e.g. Volcano, YuniKorn, Armada, MCAD, which makes it easy for users to migrate.
+But from a user perspective, the current support for batch workloads is widely different from 
+one batch system to another and requires a comprehensive migration between them. 
+It’s necessary to build a specification for batch workloads in the cloud native ecosystem, 
+and help related projects figure out their scopes and interaction. The users and communities can 
+follow this specification to work with projects in CNCF, e.g. Kubernetes, Volcano, Armada, MCAD, 
+or even out of CNCF, e.g. YuniKorn, and so on.
+  
+## Goal:
+
+The Cloud Native Batch System Initiative Working Group aims to define
+batch system specifications in the cloud native ecosystem for different batch projects,
+and related CRDs, gRPC, client libraries for projects. Those outputs will help related projects
+to figure out their scopes, and identify the different scenarios for different projects, e.g. tranditional HPC vs.
+HPC on Cloud.
+   
+## In Scope:
+
+Batch system specifications, including:
+
+- Job/ArrayJob/CronJob
+- Job Flow/Job Dependency/Job Orchestration
+- Queue
+- SchedulingPolicy (e.g. PodGroup)
+- Command Line & Script
+- Multi-Cluster Scheduling
+- Best Practices
+
+## Out Of Scope:
+
+- Non-batch related workloads
+- The implementation of the batch system
+
+## Related Communities
```

Comment:
Can you clarify how is this WG going to interact with these communities?

Reply:
We'll talk with those community to clarity the concept and APIs

Reply:
@alculquicondor are you expecting that clarification on the charter? or this is that can be worked out offline?  I'm asking to check if there's anything else add to to the charter before we merge the PR. Thx.

Reply:
I think it should be in the charter.

Reply:
There is something along those lines in our charter https://github.com/kubernetes/community/blob/master/wg-batch/charter.md#in-scope

Practically speaking, we have been reaching out to different communities to present their projects at the WG. We have had a few presentations. Some of us also hosted a Batch Day during kubecon https://events.linuxfoundation.org/kubernetes-batch-hpc-day-europe/

Reply:
Thanks, it also seem generic.

We can have something similar like shown above here.